src/model/Model_Currency.cpp

```diff
@@ -175,8 +175,9 @@ wxString Model_Currency::toString(double value, const Data* currency, int precis
 
 const wxString Model_Currency::fromString2Default(const wxString &s, const Data* currency)
 {
-    wxString str = s;
-    str.Trim();
+    // Remove any characters before the number stars. Useful especially for removing currency symbols.
+    wxString str = s.Mid(s.find_first_of("-0123456789"));
```

Comment:
Hi Vomikan

1. Why does git cop complain that "fix" is not a valid type?
2. if a number in the format of ".20" is legal the easy solution would simply be to add the decimal point to the find_first_of list
3. Since you prefer regex I'll spend some more time on that and get back to you.
Regards,